src/Integration.UnitTests/SonarAnalyzer/ActiveSolutionBoundTrackerTests.cs

```diff
@@ -95,6 +106,8 @@ public void ActiveSolutionBoundTracker_Changes()
 
             // Sanity
```

Comment:
Improved coverage of the ActiveSolutionBoundTracker Changes which respond to two events rather than one (was like before my change).